### PR TITLE
[DYNAREC_RV64] Added more opcodes

### PR DESCRIPTION
**Pull request by ksco**
for yuzu.

### CODE REVIEW

**Comment by ksco** [comment]
There are a lot more to add, which is great, I'll add them later.

File: src/dynarec/rv64/dynarec_rv64_f30f.c [hunk 1]
```diff
@@ -153,6 +168,15 @@ uintptr_t dynarec64_F30F(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
             GETGXSS_empty(v1);
             FSQRTS(v1, v0);
             break;
+        case 0x53:
+            INST_NAME("RCPSS Gx, Ex");
+            nextop = F8;
+            GETEXSS(v0, 0);
+            GETGXSS_empty(v1);
+            LUI(x3, 0x3F800); // 1.0f
+            FMVWX(v1, x3);
```
Comment (ptitSeb) [inline src/dynarec/rv64/dynarec_rv64_f30f.c:177]: warning, v1 can be the same as v0, you might need a scratch regs here

Comment (ksco) [reply src/dynarec/rv64/dynarec_rv64_f30f.c:177]: Indeed.

Comment (ksco) [reply src/dynarec/rv64/dynarec_rv64_f30f.c]: Fixed

**Pull request merged**